examples/pincell_depletion/restart_depletion.py

```diff
@@ -62,7 +62,7 @@
 time, keff = results.get_keff(time_units='d')
 
 # Obtain U235 concentration as a function of time
-uo2 = geometry.get_all_material_cells()[1]
+uo2 = geometry.get_all_material_cells()[1].fill
```

Comment:
Seemingly unrelated by maybe necessary change? This goes from selecting a `cell` to a material that occupies that cell?

Reply:
This unrelated and unnecessary, sorry about that